devel/gn-devel/Portfile

```diff
@@ -14,9 +14,12 @@ long_description    GN is a meta-build system that generates build files for Nin
 homepage            https://gn.googlesource.com/gn
 
 subport qt5-qtwebengine-gn {
-    set hyphen_version 2022-12-09
+    # Use modification date of gn sourcetree as version;
+    # see livecheck URL for whether this should be updated
+    set hyphen_version 2022-04-20
     version         [string map {"-" ""} ${hyphen_version}]
     revision        0
+    epoch           1
```

Comment:
how about we just leave it as-is and change the version to the correct value once updated/appropriate. Changing the "epoch" should IMHO only be done when absolutely required which isn't the case here. 

The change to "python311" is good though, so perhaps just increase the "revision" instead.

Reply:
> how about we just leave it as-is and change the version to the correct value once updated/appropriate. Changing the "epoch" should IMHO only be done when absolutely required which isn't the case here.

Do you mean that I specifically not add `epoch 1`? I would agree as long as livecheck works.

If you instead mean that I leave `version` as 20221209, then in that case I would rather give up on livecheck working and disable it as Marcus suggested. However it seems others might prefer abandoning the date version scheme entirely and just use e.g. the qt5-qtwebengine version instead.

Part of why I created this port using a date version which can stay behind despite further updates to qt5-qtwebengine is that actual future updates to this gn port do not seem very likely. (It is still identical to when it was created; the 20220420 update only dropped a build fix patch in favor of upstream applying it.)

> The change to "python311" is good though, so perhaps just increase the "revision" instead.
 
Python is only a build dependency, and changing the Python version does not meaningfully affect the build output, so `revision` should not need increasing.

Reply:
> However it seems others might prefer abandoning the date version scheme entirely and just use e.g. the qt5-qtwebengine version instead.

Or discontinue the qt5-qtwebengine-gn port altogether. I created it mainly so that I wasn't wasting time rebuilding gn from scratch (and without ccache) during every attempt to build qt5-qtwebengine; maybe others do not find that worthwhile.

Reply:
as I said before I see no reason to increase the epoch and set the version to an earlier dat if nothing has changed.